server/src/test/java/org/elasticsearch/action/admin/indices/create/TransportCreatePartitionsActionTest.java

```diff
@@ -60,116 +55,98 @@ public void prepare() {
+            String templateName = PartitionName.templateName(indexName);
+            assertThat(templateNames).containsOnlyOnce(templateName);
         }
     }
 
     @Test
     public void testRoutingOfIndicesIsNotOverridden() throws Exception {
-        PutIndexTemplateRequest templateRequest = new PutIndexTemplateRequest("*")
-            .patterns(List.of("*"))
-            .mapping(Map.of())
-            .settings(Settings.builder()
-                .put("number_of_shards", 1)
-                .put("number_of_replicas", 0)
-            );
-        cluster().client().execute(PutIndexTemplateAction.INSTANCE, templateRequest).get();
-
-        cluster().client().admin().indices()
-            .create(new CreateIndexRequest("index_0")
-                .settings(Settings.builder().put("number_of_shards", 1).put("number_of_replicas", 0))
-            ).get();
-        ensureYellow("index_0");
-
-        CreatePartitionsRequest request = new CreatePartitionsRequest(
-            Arrays.asList("index_0", "index_1"));
-
-        CompletableFuture<ClusterStateUpdateResponse> response = new CompletableFuture<>();
-
-        action.createIndices(request, new ActionListener<ClusterStateUpdateResponse>() {
-            @Override
-            public void onResponse(ClusterStateUpdateResponse clusterStateUpdateResponse) {
-                response.complete(clusterStateUpdateResponse);
-            }
-
-            @Override
-            public void onFailure(Exception e) {
-                response.completeExceptionally(e);
-            }
-        });
-
-        assertThat("Create indices action did not lead to a new cluster state",
-            response.get(5, TimeUnit.SECONDS).isAcknowledged(), is(true));
-
-        ClusterState currentState = cluster().clusterService().state();
-        ImmutableOpenIntMap<IndexShardRoutingTable> newRouting = currentState.routingTable().indicesRouting().get("index_0").getShards();
-        assertTrue("[index_0][0] must be started already", newRouting.get(0).primaryShard().started());
-    }
+        execute("create table table1 (a int, b int) " +
+            "partitioned by (a) " +
+            "clustered into 1 shards " +
+            "with (number_of_replicas=0)");
 
-    @Test
-    public void testCreateBulkIndicesIgnoreExistingSame() throws Exception {
-        PutIndexTemplateRequest templateRequest = new PutIndexTemplateRequest("*")
-            .patterns(List.of("*"))
-            .mapping(Map.of())
-            .settings(Settings.builder()
-                .put("number_of_shards", 1)
-                .put("number_of_replicas", 0)
-            );
-        cluster().client().execute(PutIndexTemplateAction.INSTANCE, templateRequest).get();
-        List<String> indices = Arrays.asList("index1", "index2", "index3", "index1");
-        AcknowledgedResponse response = action.execute(
-            new CreatePartitionsRequest(indices)
-        ).get();
-        assertThat(response.isAcknowledged(), is(true));
```

Comment:
Removed since `testRoutingOfIndicesIsNotOverridden` basically tests such scenario - we create some partitions, then try to insert to existing.